src/test/java/io/github/externschool/planner/controller/UserControllerTest.java

```diff
@@ -83,31 +86,11 @@ public void testPostSignupWithCorrectParameters_ReturnsUserSuccessTemplate() thr
                 "!Qwert");
 
         mockMvc.perform(MockMvcRequestBuilders.post("/signup").params(mapUser(user)))
-                .andExpect(status().isOk())
-                .andDo(print())
-                .andExpect(view().name("success"))
-                .andExpect(content().contentType("text/html;charset=UTF-8"))
-                .andExpect(content().string(Matchers.containsString("Successful Sign Up")))
-                .andExpect(model().attribute("user", equalTo(user)));
+                .andExpect(status().isFound()).andExpect(redirectedUrl("/login"));
     }
 
-    //TODO Browser-side SignUp form validation used
```

Comment:
Why do you delete this test ?

Reply:
Becous its ben created for my method

Reply:
Because its Ben created Test method for my method . I do  only one where i check that its work correct

Reply:
ok, in the future we return for this case